gui/src/3D/controls/touchHolographicMenu.ts

```diff
@@ -99,7 +99,7 @@ export class TouchHolographicMenu extends VolumeBasedPanel {
      * Creates a holographic menu GUI 3D control
      * @param name name of the menu
      */
-    constructor(name: string) {
+    constructor(name = "") {

```

Comment:
Is there a reason this `name` needs to be defined or can it just be an optional like the other constructors?